munigeo/api.py

```
@@ -349,10 +401,61 @@ def get_queryset(self):
+                    % ", ".join([x[0] for x in settings.LANGUAGES])
+                )
+        else:
+            self.lang_code = LANG_CODES[0]
+
+        if "name" in filters:
+            name = filters["name"].strip()
+            args = {"name_%s" % self.lang_code: name}
+            try:
+                postal_code_area = PostalCodeArea.objects.get(**args)
+            except PostalCodeArea.DoesNotExist:
+                raise ParseError("postalcodearea with name '%s' not found" % name)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)